src/EnergyPlus/Plant/Operation/CallingOrder.hh

```diff
@@ -0,0 +1,72 @@
+// EnergyPlus, Copyright (c) 1996-2018, The Board of Trustees of the University of Illinois,
```

Comment:
@mjwitte @EnergyArchmage One of the big organization enhancements I did on this PR was pull classes and structs out of DataPlant and into their own appropriate files.  Every time I tried to do some organization, I ended up with circular dependencies because the classes were so intermingled.  And also, every time I made a tiny change to one struct in DataPlant, it would trigger a near full rebuild of E+ because so many files depend on it.  

Now (almost) every struct is placed in its own file with license text header, and if a small change is needed, it won't automatically cause a full rebuild.  I pulled all the functions from DataPlant that shouldn't have been there in the first place, and dropped them into PlantUtilities.  I deleted at least one function that wasn't used, and I deleted a bunch of arrays and random variables in DataPlant that weren't used.

I know the changes here have become difficult to diff through, but I feel very confident that as long as the code compiles (and we aren't seeing unexpected diffs), these organizational changes were purely beneficial.  I don't intend on pulling all the components into directories, at least not here.

Reply:
I can't seem to find the new directories and little files in VS2015 solution explorer.  I reran CMAKE and successfully built the branch, but I can't find the new files in subfolders in the IDE.

Reply:
@EnergyArchmage For whatever reason, all of the source files display as a flat list in VS2015. And most of the new files are headers. I did a build and then let VS reload all and I see the new files, `Branch.hh` is in the EnergyPlusLib\Header Files folder. This will be a navigation problem soon (if not already). I'm looking for an option setting to display the list using the subfolders but I haven't found it. Maybe it's controlled by cmake?

Reply:
Thanks for starting to look at this guys.  On the Visual Studio stuff, I honestly don't have a clue.

A quick Google reveals this link shows something about changing the project files view, for a Java project, but I don't expect it to be helpful: https://msdn.microsoft.com/en-us/library/aa265177(v=vs.60).aspx

A little more searching reveals Visual Studio doesn't want to do anything with the hierarchy.  Though there was also this https://stackoverflow.com/a/28702616 which describes a CMake "source_group".  Might be worth trying on someone's machine locally to see if they show up in folders.  @kbenne any thoughts on this front?

Reply:
Discovered that what looks like folders in the VS solution explorer are *filters*. \Build\src\EnergyPlus\energypluslib.vcxproj.filters is an xml file that looks like this:
```
<?xml version="1.0" encoding="UTF-8"?>
<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <ItemGroup>
    <ClCompile Include="D:\MJW-Work2\EnergyPlus Git2\EnergyPlus-ReviewBranch4\src\EnergyPlus\AirflowNetworkBalanceManager.cc">
      <Filter>Source Files</Filter>
    </ClCompile>
    <ClCompile Include="D:\MJW-Work2\EnergyPlus Git2\EnergyPlus-ReviewBranch4\src\EnergyPlus\AirflowNetworkSolver.cc">
      <Filter>Source Files</Filter>
```
Haven't figured out what builds this file.

Reply:
I was looking for enums.cc, branch.cc, etc but now I realize the file breakout was just header files.

Reply:
Ahh makes sense. Sorry for the confusion. Yeah it was pretty much all header stuff. If it ever broke into more implementation files they'd go in there too, but for now it's just headers.

Reply:
@Myoldmopar @mjwitte Sorry to be a wet blanket, but I'm just really not a fan of this type of organization. I get that we don't want lots of files in one directory like certain other projects (cough cough), so one layer of organizing directory structure makes sense. But this to me is going too far, and is going to be an IDE nightmare.

Reply:
What other projects?

![image](https://user-images.githubusercontent.com/849824/37104439-573baff6-21f2-11e8-8f2d-ee24e7201ce2.png)

One layer of organizing seems good.  I can pull them all into the single `Plant/` subdirectory and name the files with more descriptive names.  Those two fixes should satisfy all of you wet blankets and Visual Studio users :upside_down_face:.